src/components/EpisodesControls/EpisodesControls.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from "react";
+import { data } from "COMPONENTS/Episodes/";
+import { RENDER } from "./";
+import { Episodes } from "COMPONENTS/Episodes/";
+
+export default function EpisodesControls(){
+	
+	//CONTEXT
+	//---------------------------
+	const { activeIndex } = useContext(Episodes);
+
+	//RENDER
+	//---------------------------
+	const pagination = data.map(RENDER.pagination);
+
+	return (
+		<nav>
+			<button aria-label="Previous episode." />
+			<button aria-label="Next episode." />
```

Comment:
in addition to my comment from the other branch (these buttons should have `aria-controls` for the `<ol>` of `episodes`, it occurrs to me that they also control the pagination below as well (ie. which one has `aria-current`)

Reply:
Added controls for episodes list.
TODO: using context to set aria-current and then looking at controls with that also.

Reply:
Controls now point to the episode list and the pagination list.
Aria-current determined by activeindex.